internal/server/polling/config.go

```diff
@@ -0,0 +1,67 @@
+package polling
+
+import (
+	"context"
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const DefaultConfigMapName = "default/branch-based-planner"
+
+// Example ConfigMap
+//
+// The secret is a reference to a secret with a 'token' key.
+//
+// ---
+// apiVersion: v1
+// kind: ConfigMap
+// metadata:
+//   name: branch-based-planner
+// data:
+//   # Secret to use to use GitHub API.
+//   # Key in the secret: token
+//   secretNamespace: flux-system
```

Comment:
Is there a good reason to have a cross-namespace reference here? (Apologies if it's in a design doc somewhere and I missed it)
Usually, allowing cross-namespace references represents a vulnerability.

Reply:
OK, sounds like it's provisional. It'd be good to record the things that are not intended to remain standing, maybe in comments and the PR description.